static/templates/invitation_failed_error.hbs

```diff
@@ -1,6 +1,13 @@
-<p>{{ error_message }}</p>
+<p id="err_msg">{{ error_message }}</p>
 <ul>
     {{#each error_list}}
     <li>{{this}}</li>
     {{/each}}
 </ul>
+{{#if is_invitee_deactivated}}
+    {{#if is_admin}}
+    <p>{{t "You can reactivate deactivated users from "}}<a href="#organization/deactivated-users-admin">{{t "organization settings."}}</a></p>
```

Comment:
This type of i18n construction doesn't work for languages with a different word order than English.  See https://zulip.readthedocs.io/en/latest/translating/internationalization.html